open_schools_platform/student_management/students/models.py

```diff
@@ -37,7 +37,7 @@ class StudentProfile(BaseModel):
         blank=True,
         null=True,
     )
-    photo = models.ImageField(storage=ClientDocsStorage(), default=None, blank=True, null=True)
+    photo = models.ForeignKey(Photo, on_delete=models.SET_NULL, null=True, related_name="photo", blank=True)
```

Comment:
can you explain why you choose this options for photo?

Reply:
on_delete=models.SET_NULL because if we delete photo, we don't want StudentProfile to be deleted, we just want photo field to be None. StudentProfile can exist without photo.